### ISSUE TITLE
possible to add an extra option

### ISSUE THREAD

**Issue by pieterlabie** [Nyr/openvpn-install#104]
Hi, 

I have been using this script for a wile now . 
I was thinking of its possible to add a function for not routing everything true the tunnel . 
So i can access the home network without pushing everything over it . 

Best regards

**Comment by Nyr** [comment]
Not planned.

You can remove the `redirect-gateway def1` and iptables postrouting rule to achieve this if you want to.

**Comment by pieterlabie** [comment]
Hi thanks for the info but where do i delete the redirect-gateway def1 ? 
Im not that good with it :)

**Comment by Nyr** [comment]
`/etc/openvpn/server.conf`

A simple Google search would had been enough to find this information.

**Comment by renato** [comment]
I found this script and it's awesome, made my life so much easier with OpenVPN.

However, I think this could be an option, too.

I installed my first server with this script not knowing that all client traffic would go through the VPN.

No problem, I read the OpenVPN docs, then removed the server pushs (`redirect-gateway` and `dhcp-option`) and it fixed the traffic.

But I missed the `iptables` rule, so the server stopped seeing the client. Took me a while to figure it out, and I only did it after reading this issue.

I'm to blame for missing the "undo" steps, but I think it could be an option or just a note in the README so future users can know what's happening and how to revert it.
